source/features/add-updated-date-to-pinned-issues.tsx

```diff
@@ -0,0 +1,69 @@
+import clockIcon from 'octicon/clock.svg';
+import features from '../libs/features';
+import * as api from '../libs/api';
+import {getRepoGQL} from '../libs/utils';
+import React from 'dom-chef';
+import select from 'select-dom';
+
+async function init(): Promise<false | void | any> {
+	// Get issues links that don't already have a specific sorting applied
+	for (const link of select.all<HTMLAnchorElement | any>('.pinned-issue-item')) {
+		const issueNUmber: any = Number((select<any>('a.link-gray-dark.no-underline.h4[href]', link)).href.split('/').slice(-1).pop());
+		const [lastCommented, lastUpdated] = await getLastCommented(issueNUmber);
```

Comment:
Instead of making multiple API requests, look at how some other features make a single request, search `buildQuery` in our code

Reply:
This is still pending